doc/HOWTO-DAEMON.md

```diff
@@ -0,0 +1,159 @@
+# Daemon mode
+
+In the "daemon mode", Sonar stays memory-resident and pushes data to a sink.  In this mode, the only
+command line parameter is the name of a config file.
+
+The daemon is a multi-threaded system that performs system sampling, communicates with the sink, and
+handles signals and lock files.
+
+## CONFIG FILE
+
+The config file is an ini-type file.  Blank lines and lines starting with '#' are ignored.  Each
+section has a `[section-name]` header on a line by itself.  Within the sections, there are
+`name=value` pairs where names are simple identifiers matching `/[a-zA-Z_][-a-zA-Z_0-9]*/` and values
+may be quoted with `'`, `"`, or "`"; these quotes are stripped.  Blanks before and after names and
+values are stripped.
+
+Boolean values are `true` or `false`.  Duration values express a time value using the syntax `__h`,
+`__m`, or `__s`, denoting hour, minute, or second values (uppercase HMS also allowed); values must
+be nonzero. For cadences, second values must divide a minute evently and be < 60, minute values must
+divide an hour evenly and < 60, and hour values must divide a day evenly or be a positive multiple
+of 24.  (Some sensible cadences such as 90m aka 1h30m are not currently expressible.)
+
+The config file has `[global]` and `[debug]` sections that control general operation; a section for
+the transport type chosen, currently only `[kafka]`; and a section each for the sonar operations,
```

Comment:
Oops, "kafka" here needs to be taken out, something else in its place; then "kafka" added later with that patch.